docs/config/routes.md

```
@@ -159,39 +158,39 @@ gordon reload
 
 ```toml
 [routes]
-"app.local" = "myapp:latest"
-"api.local" = "myapi:latest"
+"dev-app.example.com" = { image = "myapp:latest" }
+"dev-api.example.com" = { image = "myapi:latest" }
 ```
 
 Add to `/etc/hosts`:
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)